index.html

```diff
@@ -189,16 +190,38 @@ <h5 class="modal-title">Input New Employee</h5>
     </div>
 
 
-    <div class="offcanvas offcanvas-bottom" tabindex="-1" id="offcanvasBottom" aria-labelledby="offcanvasBottomLabel">
+    <div class="offcanvas offcanvas-end" tabindex="-1" id="offcanvasRight" aria-labelledby="offcanvasRightLabel">
         <div class="offcanvas-header">
-            <h5 class="offcanvas-title" id="offcanvasBottomLabel">Offcanvas bottom</h5>
+            <h5 class="offcanvas-title" id="offcanvasRightLabel">Offcanvas right</h5>
             <button type="button" class="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"></button>
         </div>
-        <div class="offcanvas-body small">
-            ...
+        <div class="offcanvas-body">
+            <h2> Assign Tasks </h2>
+
+            <div>
+                <select id="daySelect" class="form-select">
```

Comment:
not sure that we shoudl have a day dropdown... I could be talked into it but seems like it will overcomplicate stuff?